AY2021S2-CS2103-T14-1/tp
# Docs: Update landing page and README

## What this does
This PR:
1. Updates the landing page to use an updated product image, and updated the acknowledgements
2. Updates the github README with the same content

Closes #131 

## How to test
1. cd to `docs/` folder of the project.
2. Run `bundle exec jekyll serve`.
3. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
4. Check that the landing page has the updated product image and the content overall makes sense.
5. Check that the sitemap, acknowledgements and license links work.
